### PR TITLE
Further distill the scripts

### PR DESCRIPTION
**Pull request by dr3**
1) `cypress:interactive` had a typo
2) Nobody runs `killApp` manually, so make it a script
3) Combine `test:lint` and `test:lint:css` cus nobody running `test:lint:css`
4) Update readme for new e2e commands

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

File: scripts/killApp.sh [hunk 1]
```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
```
Comment (sareh) [inline scripts/killApp.sh:1]: You should add a comment to the top of this file stating that this script should only be run locally and never on CI.

**Comment by andrew-nowak** [comment]
Not sure why killApp 'should only be run locally, not on CI'? `test:ci` is the script which is going to leave a server running on 7080 after it finishes, and (personally) when running locally I would prefer to see errors that a server is running on 7080 and left to resolve it myself, rather than having a script automatically kill it

IMO the place where killApp would be most useful is as the final command in `test:ci` (cleaning up the server when we're finished with it)

File: README.md [hunk 2]
```diff
@@ -256,13 +256,13 @@ Cypress .visit() function is locked to visiting a single domain per test. This b
 Here is an example command:
 
 ```
-CYPRESS_APP_ENV=test CYPRESS_UK=true npm run cypress:interactive
+CYPRESS_APP_ENV=test CYPRESS_UK=true npm run cypress
```
Comment (jamesbhobbs) [inline README.md]: Can we also mention the cypress smoke variable?

**Review by jamesbhobbs — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (dr3) [reply README.md]: Done :)

**Comment by dr3** [comment]
@aacn500 on jenkins you just get `-bash: lsof: command not found` :(

**Review by jamesbhobbs — APPROVED** [review]

**Review by oluoluoxenfree — APPROVED** [review]

**Pull request merged**